bundles/org.openhab.binding.opensprinkler/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -109,17 +167,51 @@
 		<category>Switch</category>
 	</channel-type>
 
+	<channel-type id="signalStrength">
+		<item-type>Number:Power</item-type>
+		<label>Signal Strength</label>
+		<description>Wi-Fi Received Signal Strength Indicator.</description>
+		<category>QualityOfService</category>
+		<state readOnly="true" pattern="%d %unit%"></state>
+	</channel-type>
+
 	<channel-type id="remainingWaterTime">
 		<item-type>Number:Time</item-type>
 		<label>Remaining Water Time</label>
 		<description>Read-only property of the remaining water time of the station.</description>
+		<category>Time</category>
 		<state readOnly="true" pattern="%.0f min"/>
 	</channel-type>
 
 	<channel-type id="nextDuration">
 		<item-type>Number:Time</item-type>
-		<label>Next Open Duration</label>
+		<label>Next Duration</label>
 		<description>The duration the station will be opened the next time it is switched on.</description>
+		<category>Time</category>
 		<state readOnly="false" pattern="%.0f min"/>
 	</channel-type>
+
+	<channel-type id="rainDelay">
+		<item-type>Number:Time</item-type>
+		<label>Rain Delay</label>
+		<description>The amount of time in hours to delay the running of any programs.</description>
```

Comment:
"of any program." without an s at the end?

Reply:
DONE, change made.